site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -1,81 +1,92 @@
-                    {% if not gradeable.isTeamAssignment() %}
-                        <input type="checkbox" class="ignore" name="submit-before-grading" id="submit-before-grading" value="Submit Before Grading">Submit Before Grading</input>
-                    {% endif %}
-                    <div class="btn-container" >
-                        <input type="text" id="number_to_peer_grade" placeholder="Number of Students">
-                        <input type="checkbox" id="all_grade" name="all_grade" class= "ignore" value="All Grade All"> All Grade All
-                        <input type="button" name="random_peer_graders_list" class="btn btn-primary key_to_click" id="random_peer_graders_list" tabindex="0" value="Generate"></input>
-                    </div>
-                </div>
+        </div>
+        <div class="col-md-6 pr-md-0">
+            <div>
+                Enter number of peers for one student to grade and randomly assign peer grading in class.
+                <hr>
+                <input type="radio" id="all_grade" name="all_grade" class= "ignore" value="All Grade All" checked> All Grade All<br>
+                <input type="radio" name="all_grade" class="ignore" value="Number of Students">
+                <input type="number" min="1" id="number_to_peer_grade" placeholder="Number of Students">
+                <br>
+                <br>
+                <input type="checkbox" class="ignore" name="restrict-to-registration" id="restrict-to-registration" value="Restrict to Registration Sections">Restrict to Registration Sections
+                {% if not gradeable.isTeamAssignment() %}
+                    <br><input type="checkbox" class="ignore" name="submit-before-grading" id="submit-before-grading" value="Submit Before Grading">Submit Before Grading
+                {% endif %}
+                <br><br>
+                <input type="button" name="random_peer_graders_list" class="btn btn-primary key_to_click" id="random_peer_graders_list" tabindex="0" value="Generate">
             </div>
         </div>
-        <div class="row">
-            <div class="card col-md-12" style="margin:0px">
-                <div class="content"style="margin:0px">
-                    <p>Other Controls: </p>
-                    <div class="btn-wrapper" >
-                        <input type="button" name="clear_peer_matrix" class="btn btn-default key_to_click ignore" id="clear_peer_matrix" tabindex="0" value="Clear Peer Matrix"></input>
-                         <input type="button" name="download_peer_csv" class="btn btn-default key_to_click ignore" id="download_peer_csv" tabindex="0" value="Download Peer CSV" onclick="downloadPeerDataAsCsv('peer-table')"></input>
-                    </div>
-                    <br>
+    </div>
+    <br>
```

Comment:
Not asking for a change on this as it's not a huge thing, but if we wanted to strictly follow best accessibility / web practices, we should only use the `<br>` tag for line breaks in text where the break is significant (e.g. if writing a poem or an address). We should __not__ be using them to put margins between elements, rather relying on CSS `margin` to do this. Similar, for `<input>` tags we want on individual lines should be wrapped in a `<div>`. While it'd be nice to fix this, if it's a non-trivial amount of effort, fine to not deal with this.